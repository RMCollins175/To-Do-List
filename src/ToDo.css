.ToDo-Done {
    text-decoration: line-through;
}

.ToDo-NotDone {
    text-decoration: none;
}

