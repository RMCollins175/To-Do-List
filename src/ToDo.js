import React from "react";
import "./ToDo.css";

class ToDo extends React.Component {


  render() {

    return (
      <div className="ToDo">
        <div>
          <li className={this.props.isDone ? "ToDo-Done" : "ToDo-NotDone"} onClick={this.props.toggleToDo}>
            {this.props.item}
            <button onClick={this.props.remove}>X</button>
          </li>
        </div>
      </div>
    );
  }
}

export default ToDo;


// className={this.props.done ? "ToDo-done" : "ToDo-normal"}