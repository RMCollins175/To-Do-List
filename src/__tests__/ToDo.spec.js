import React from 'react'
import ToDo from '../Todo'
import { create } from 'react-test-renderer'


describe("Todo component", () =>{
    test("Mathches the snapshot", () => {
        const todo = create(<ToDo />);
        expect(todo.toJSON()).toMatchSnapshot();
    })
})

