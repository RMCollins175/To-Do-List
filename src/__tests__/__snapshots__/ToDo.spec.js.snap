// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo component Mathches the snapshot 1`] = `
<div
  className="ToDo"
>
  <div>
    <li
      className=""
    >
      <button>
        X
      </button>
    </li>
  </div>
</div>
`;
